jdi-light-html-tests/src/test/java/io/github/epam/example/common/LinkExampleTests.java

```diff
@@ -43,6 +45,13 @@ public void linkTest() {
                 .css("font-size", is("14px"));
     }
 
+    @Test
+    public void clickLinkTest() {
+        githubLink.click();
+        assertEquals(getUrl(), EXPECTED_URL);
+        html5Page.open();
```

Comment:
For what purpose does html5Page open again?

Reply:
I didnt't notice that it used  @BeforeMethod, not @BeforeClass when it is checked that html5 page open.
So there is no purpose for opening html5Page again ->fixed it